dpsim-villas/examples/cxx/FpgaExample.cpp

```diff
@@ -174,17 +190,21 @@ int main(int argc, char *argv[]) {
   CommandLineArgs args(argc, argv, "FpgaExample", 0.01, 10 * 60, 5.);
   CPS::Logger::setLogDir("logs/" + args.name);
 
-  auto intf = std::make_shared<InterfaceVillas>(buildFpgaConfig(args));
+  auto intf = std::make_shared<InterfaceVillasQueueless>(
+      buildFpgaConfig(args), "FpgaExample", spdlog::level::off);
   auto logger = DataLogger::make(args.name);
 
-  auto sys = getTopology(args, intf, logger);
+  auto sys = getTopology(args, intf, nullptr);
 
-  RealTimeSimulation sim(args.name, args);
+  Simulation sim(args.name, args);
   sim.setSystem(sys);
   sim.addInterface(intf);
-  sim.addLogger(logger);
+  //sim.addLogger(logger);
   sim.run();
 
+  CPS::Logger::get("FpgaExample")->info("Simulation finished.");
+  sim.logStepTimes("FpgaExample");
+
   //std::ofstream of("task_dependencies.svg");
```

Comment:
Please remove dead-code or make it configurable

Reply:
This is there to stay consistent with the other examples.